apisix/plugins/request-id.lua

```diff
@@ -41,9 +72,134 @@ function _M.check_schema(conf)
 end
 
 
+-- Generates the current process worker number
+local function gen_worker_number(max_number)
+    if worker_number == nil then
+        local etcd_cli, prefix = core.etcd.new()
+        local prefix = prefix .. "/plugins/request-id/snowflake/"
+        local uuid = uuid.generate_v4()
+        local id = 1
+        while (id <= max_number) do
+            ::continue::
```

Comment:
continue back may causes the id > max_number. For instance:
```
local id = 0
while id <= 3 do
    ::continue::
    if id == 3 then
        print(id)
        id = id + 1
        goto continue
    end
    print(id)
    id = id + 1
end
-- 0, ..., 4
```

Reply:
Indeed, I will fix it